torch/csrc/inductor/aoti_torch/c/shim.h

```diff
@@ -152,6 +152,25 @@ AOTI_TORCH_EXPORT AOTITorchError aoti_torch_create_tensor_from_blob(
     AtenTensorHandle* ret // returns new reference
 );
 
+AOTI_TORCH_EXPORT AOTITorchError aoti_torch__scaled_dot_product_flash_attention(
+    AtenTensorHandle query,
+    AtenTensorHandle key,
+    AtenTensorHandle value,
+    double dropout_p,
+    bool is_causal,
+    bool return_debug_mask,
+    double scale,
+    AtenTensorHandle* ret0, // returns new reference
+    AtenTensorHandle* ret1, // returns new reference
+    AtenTensorHandle* ret2, // returns new reference
+    AtenTensorHandle* ret3, // returns new reference
+    int64_t* ret4,
+    int64_t* ret5,
```

Comment:
dynamic shapes?!

Reply:
Those two fields stand for max_seqlen_batch_q and max_seqlen_batch_kv, which are returned from the sdpa kernel:

https://github.com/pytorch/pytorch/blob/36eb1bb5486e6899a806cbe82c42c1021d456e3e/aten/src/ATen/native/nested/cuda/NestedTensorTransformerFunctions.cpp#L245-L246

I think they can be dynamic given they are related to sequence length and batch?